### PR TITLE
Update to 2021.01.0

### PR DESCRIPTION
**Pull request by aleb**

### CODE REVIEW

File: org.pitivi.Pitivi.json [hunk 1]
```diff
@@ -55,16 +55,29 @@
             "no-autodownload": true
         }
     },
-    "modules" : [
+    "modules": [
         "python3-matplotlib.json",
         {
-            "name" : "gsound",
-            "sources" : [
+            "name": "libcanberra",
```
Comment (aleb) [inline org.pitivi.Pitivi.json:61]: We want to get rid of gsound but until we can, libcanberra is required by gsound.

File: org.pitivi.Pitivi.json [hunk 2]
```diff
@@ -85,63 +98,56 @@
             ]
         },
         {
-            "name" : "libopus",
```
Comment (aleb) [inline org.pitivi.Pitivi.json:88]: libopus is in the 3.38 runtime already.

**Comment by thiblahute** [comment]
Lgtm, thx for working on that!

**Comment by aleb** [comment]
Pitivi depends on matplotlib. Matplotlib depends on numpy. Numpy needs setuptools<49.2.0 to be built:

```
python3 ../../flatpak-builder-tools/pip/flatpak-pip-generator --runtime org.gnome.Sdk/x86_64/3.38 "setuptools<49.2.0" numpy matplotlib
```

The runtime already has setuptools 51.3.3:
```
$ flatpak --devel --share=network --filesystem=/tmp --command=bash run org.gnome.Sdk/x86_64/3.38                               
[📦 org.gnome.Sdk ~]$ pip3 list | grep setuptools
setuptools 51.3.3.post20210206
```

Unfortunately pip cannot install two versions of a package at the same time:
https://stackoverflow.com/questions/6570635/installing-multiple-versions-of-a-package-with-pip

This means to install the older setuptools version required by numpy the existing one needs to be uninstalled, but this cannot be done:
https://flathub.org/builds/#/builders/19/builds/4620:
```
========================================================================
Building module python3-setuptools in /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/python3-setuptools-1
========================================================================
]2;flatpak-builder: Building python3-setuptools]2;flatpak-builder: Installing python3-setuptoolsRunning: pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "setuptools<49.2.0"
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/python3-setuptools --nofilesystem=host --filesystem=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/python3-setuptools-1 --bind-mount=/run/build/python3-setuptools=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/build/python3-setuptools-1 --build-dir=/run/build/python3-setuptools --bind-mount=/run/ccache=/srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1613204546 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection ' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection ' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed  -L/app/lib/codecs/lib' --env=PYTHON=python3 --env=FREI0R_PATH=/app/lib/frei0r-1/ --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0/:/app/lib/codecs/lib/gstreamer-1.0/ --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib:/app/lib/codecs/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=48 /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-jGfiWE /bin/sh -c 'pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "setuptools<49.2.0"'
Looking in links: file:///run/build/python3-setuptools
Processing ./setuptools-49.1.3.zip
Building wheels for collected packages: setuptools
  Building wheel for setuptools (setup.py) ... 
done
[?25h  Created wheel for setuptools: filename=setuptools-49.1.3-py3-none-any.whl size=789532 sha256=7673c8c53de54cdb75e5ede15cabc61006113a4b594efe1d7392d6eae184d7cd
  Stored in directory: /srv/buildbot/.cache/pip/wheels/05/82/f9/8b9fd88c0c7008e075882686353456018e19bf17410142df9f
Successfully built setuptools
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 51.3.3.post20210206
    Uninstalling setuptools-51.3.3.post20210206:
ERROR: Could not install packages due to an OSError: [Errno 30] Read-only file system: '/usr/bin/easy_install-3.8'
```

This is all because the `.whl` files are not accepted because we need to build from source to be able to build for aarch64.

**Comment by hfiguiere** [comment]
the `version` for the audio plugin extensions must be upgraded to 20.08 for the gnome 3.38 runtime. line 51 in th manifest.

**Comment by hfiguiere** [comment]
also why disabling aarch64?

**Comment by hfiguiere** [comment]
> also why disabling aarch64?

:-(

**Comment by aleb** [comment]
> also why disabling aarch64?

I explained above why building for aarch64 does not work. Let's merge this as is and come back to aarch64 afterwards?

See commit https://github.com/flathub/org.pitivi.Pitivi/commit/689ea152338aa64edaf35de5fbd7f2d6f52422ff (and its parent) for the ideal python3-matplotlib.json building from source which fails. See the build failure above following this "689ea15" commit.

**Comment by hfiguiere** [comment]
This will close #20 as well.

**Comment by hfiguiere** [comment]
This was closed with #22 so it can be closed too (includes all from this)

**Pull request closed without merging**